test/test_dao.py

```diff
@@ -732,3 +732,24 @@ def test_get_all_users(self):
         self.assertEquals(user.full_name, self.user_full_name_2)
         self.assertEquals(user.admin_regions, self.user_admin_regions_2)
 
+    def test_get_and_insert_pending_merge(self):
```

Comment:
Can you add a test to see that getting a merge with an ID that doesn't exist returns None?